src/shell/shell_misc.cpp

```diff
@@ -50,6 +51,15 @@ static void outc(Bit8u c) {
 	DOS_WriteFile(STDOUT,&c,&n);
 }
 
+static void backone() {
+	BIOS_NCOLS;
```

Comment:
If`BIOS_NCOLS` is (only) needed to give us `ncols`, suggest moving this just before we use `ncols`.

Reply:
Good suggestion. Done.